content/en/infrastructure/cloud_cost_management.md

```diff
@@ -85,7 +95,23 @@ Attach the new S3 policy to the Datadog integration role.
 3. Click **Attach policies**.
 4. Enter the name of the S3 bucket policy created above.
 5. Click **Attach policy**.
+
+## Investigating your costs
+
+Navigate to your costs within Datadog by clicking **Infrastructure > Cloud Costs** in the UI. This view lets you quickly filter and group your spending to hone in on what’s changing over time, answering questions such as:
+
+- Which teams and applications are responsible for changes in spend?
+- What cloud products and usage types (i.e. EC2 Data Transfer) are driving the increase in spend?
+- How effective is the coverage and utilization of my discount plans?
+
+{{< img src="infrastructure/cloudcost/cloud_cost_analyze_page.png" alt="Slice and dice your costs by tag using the Cloud Cost UI"  >}}
+
+## Cloud costs in dashboards
+
+Visualizing infrastructure spend alongside related utilization metrics can help you spot potential inefficiencies and savings opportunities. You can add cloud costs to widgets in Datadog dashboards by selecting the *Cloud Cost* data source.
  
+{{< img src="infrastructure/cloudcost/cloud_cost_data_source.png" alt="Cloud Cost available as a data source in dashboard widget creation"  >}}
+
 ## Cost types
  
 You can visualize your ingested data using the following cost types:
```

Comment:
TODO: Need more detailed metric descriptions and easy to understand examples

Reply:
Perhaps we can just quote the examples linked here? https://aws.amazon.com/blogs/aws-cloud-financial-management/understanding-your-aws-cost-datasets-a-cheat-sheet/

Reply:
I'd definitely recommend adding that link